Dev/luke

lukeamyers opened the pull request:
### modifications to eos_fit.py
plot_config_energy(): makes a graph of energy vs configuration with configs organized from lowest energy to highest

plot_config_histogram(): makes a histogram of the energy data

amkrajewski reviewed (approved):
I trust Nigel and Yi Wang on this one for now. All changes will be later reviewed by people familiar with DFTTK before merging this test branch to the PRL's main branch.

lukeamyers committed:
two new functions in eos_fit.py plot_config_energy() and plot_energy_histogram()

lukeamyers committed:
Merge pull request #17 from lukeamyers/dev/temp

two new functions in eos_fit.py plot_config_energy() and plot_energy_…

lukeamyers committed:
new functions in cstdn.py for df manipulation

lukeamyers committed:
set_up_ev_from_fixed_volume_calculations()

funtion to help bridge the results from single fixed volume calculations to runnning ev curves

lukeamyers committed:
Changed get_lowest_atomic_energy_configs()
turns out it's really simple with pandas, so this function is not really necessary.
